clc/modules/object-storage/src/main/java/com/eucalyptus/objectstorage/pipeline/binding/ObjectStorageGETBinding.java

```diff
@@ -88,16 +88,13 @@
     newMap.put(BUCKET + HttpMethod.GET.toString() + ObjectStorageProperties.BucketParameter.marker.toString(), "ListBucket");
```

Comment:
Nothing to change here, this was a heads-up of a change to this area of the code since this branch code was written.